source/functions/mongodb/api.txt

```diff
@@ -6,6 +6,18 @@ MongoDB API Reference
 
 .. default-domain:: mongodb
 
+.. meta:: 
+   :keywords: code example
+   :description: Learn how to use Functions for querying using MongoDB's Atlas API. 
```

Comment:
```suggestion
   :description: Learn how to use Functions to query a MongoDB Atlas data source.
```

Your description is technically correct. This phrasing avoids the double "ing" + "Atlas API" on its own kind of makes me think of the Atlas Admin API instead of the "Query API"/MQL.